### DISCUSSION TITLE
httprunner4中，字符串中的“$${A}$B”里的第一个$符号会自动消失，如何避免

### DISCUSSION THREAD

**Discussion by Chenziyi999** [httprunner/httprunner#1738] [2 - 答疑（Q&A）]
说明：
${A}是定义的变量 =>  A = 11

存在于.with_json()中，格式大致如下
{
“AAA” ："$${A}$B"
}

问题：
第一个$符号，在请求中会自动消失，这个问题如何避免.
请求结果会变为
{
“AAA” ："11$B"
}
我期望得到的结果是
{
“AAA” ："$11$B"
}

**Comment by dragonleeee** [comment]
前面在加一个$
